api/v1beta1.go

```diff
@@ -42,6 +42,10 @@ func setupV1Beta1AssetRoutes(baseURL string, router *mux.Router, ah *handlers.As
 		Methods(http.MethodPut, http.MethodHead).
 		HandlerFunc(ah.Upsert)
 
+	router.Path(baseURL).
+		Methods(http.MethodPatch, http.MethodHead).
```

Comment:
any specific reason we use `HEAD` here?

Reply:
was being brought from legacy code and copypasted over from other routes, wasn't really sure the first reason we had that.

🤔  Should we remove it from all routes?

Reply:
@StewartJingga afaik `HEAD` is a short version of `GET`, usually it is being used for resource/system health checking (figuring out whether the docs exist or not). For example, in Elasticsearch afaik before the client connect to the cluster, it sends HEAD first.

Even if we want to keep it, would be more make sense if we keep it in `GET` only. I think we need to remove it from the others PUT/PATCH.

Since we are not really have a behaviour of `HEAD` in our API, removing all of them would be better.